themes/default/content/docs/pulumi-cloud/webhooks.md

```diff
@@ -174,9 +176,9 @@ The following table describes the various event filters available and the contex
 When creating a webhook, you can choose between the generic JSON webhook payload or `slack`
 formatted events.
 
-### Slack-formatted Webhooks
+### Slack Webhooks
 
-Slack-formatted webhooks allow you to seamlessly integrate notifications about your Pulumi stacks and organizations
+Webhooks formatted for Slack allow you to seamlessly integrate notifications about your Pulumi stacks and organizations
```

Comment:
The "formatted" part seems like an implementation detail. I'd probably just mirror what we call them generally and in the UI.

```suggestion
Slack Webhooks allow you to seamlessly integrate notifications about your Pulumi stacks and organizations
```